.travis.yml

```diff
@@ -62,7 +67,7 @@ script:
           export DISPLAY=:99.0;
           /sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_99.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -ac -screen 0 1024x768x24;
           
-          docker run wesnoth-repo:16.04 bash -c './docker_run.sh "$@"' bash "$NLS" "$TOOL" "$CC" "$CXX" "$CXXSTD" "$EXTRA_FLAGS_RELEASE" "$WML_TESTS" "$WML_TEST_TIME" "$PLAY_TEST" "$MP_TEST" "$BOOST_TEST";
+          docker run -v `pwd`/build-cache:/home/wesnoth-travis/build wesnoth-repo:16.04 bash -c './docker_run.sh "$@"' bash "$NLS" "$TOOL" "$CC" "$CXX" "$CXXSTD" "$EXTRA_FLAGS_RELEASE" "$WML_TESTS" "$WML_TEST_TIME" "$PLAY_TEST" "$MP_TEST" "$BOOST_TEST";
```

Comment:
My guess would be no, since the full path is `/home/travis/build/wesnoth/wesnoth/build-cache` and I'd assume that `~` would evaluate to just `/home/travis`, but I didn't actually try it.

Reply:
Is "`pwd`/" really needed? A relative path should work as well?
If it's really needed you could as well use "$PWD"/.

Reply:
It does have to be an absolute path.  I have no particular preference for either though, so I can change it if you'd prefer `$PWD`.